web/controllers/api/package_controller.ex

```diff
@@ -6,7 +6,7 @@ defmodule HexWeb.API.PackageController do
 
   def index(conn, params) do
     page     = HexWeb.Utils.safe_int(params["page"])
-    search   = HexWeb.Utils.safe_search(params["search"])
+    search   = params["search"]
```

Comment:
We still need the conversion to `nil` for `""`. Maybe add a new function to Package that only does the String.replace?